plugins/cost-insights/src/utils/change.ts

```diff
@@ -45,11 +47,29 @@ export function growthOf(ratio: number, amount?: number) {
 export function getComparedChange(
   dailyCost: Cost,
   metricData: MetricData,
+  duration: Duration,
 ): ChangeStatistic {
   const ratio = dailyCost.change.ratio - metricData.change.ratio;
-  const amount = dailyCost.aggregation.slice().sort(aggregationSort)[0].amount;
```

Comment:
Oof, how did we miss this? Good catch, it should be the sum rather than just a single daily value.